data.sql

```diff
@@ -1,5 +1,8 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+    ('Agumon', '2020-02-03', 0, true, 10.5),
+    ('Gabumon', '2018-11-20', 2, true, 8),
+    ('Pikachu', '2021-01-07', 1, false, 15.04),
+    ('Devimon', '2017-05-12', 5, true, 11);
```

Comment:
- [ ] To match the project's requirements, kindly make sure you use the correct provided data as your current ones appear to be be different. The `date_of_birth` for `Gabumon` should be `2018-11-15` and not `2018-11-20`